.markdownlint.json

```diff
@@ -0,0 +1,4 @@
+{
+    "code-block-style": false,
+    "commands-show-output": false
```

Comment:
i think this is not correct. see the `aya-tool` example which does show its output. Just remove the `$` signs where the linter complains.

Reply:
This does not stop anyone from using the dollar sign in order to show the output.
This is just the linter config - it disables the lint which tells you to remove the dollar sign in unneeded places, which seems to be a convention in this repo.

Reply:
I agree with disabling this lint, it makes no sense to me (enforcing the lack of `$`) and the name is misleading.

I would say "please add the comment", but JSON files don't support comments, FML...

Reply:
It does make some sense, the rationale is given: https://github.com/updownpress/markdown-lint/blob/master/rules/014-commands-show-output.md

Reply:
Your repo your rules :P
Pushed a fix.